src/main/java/frc/robot/RobotContainer.java

```diff
@@ -44,75 +46,144 @@ public class RobotContainer {
+        driverControllerXButton.whileTrue(
+          new ConditionalCommand(
+            new PrintCommand("Arm inside robot"),
+            new DropCone(intake), 
+            () -> armIsInsideRobot()
+        ));
+        driverControllerBButton.whileTrue(
+          new ConditionalCommand(
+            new PrintCommand("Arm inside robot"),
+            new IntakeCone(intake), 
+            () -> armIsInsideRobot()
+        ));
+    }
 
     //operator
     {
         //floor arm
-        operatorControllerLeftJoystickButton.onTrue(new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_INSIDE_ROBOT));
-        operatorControllerRightBumper.onTrue(
+        operatorControllerYButton.onTrue(
           new SequentialCommandGroup(
             new MoveArmToPosition(arm, Constants.ARM_INSIDE_ROBOT),
-            new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_OUTSIDE_ROBOT)
-          ));
-
-        //floor intake
-        operatorControllerRightBumper.whileTrue(new FloorPickup(floorIntake));
-        operatorControllerLeftBumper.whileTrue(new FloorDrop(floorIntake));
+            new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_GROUND_INTAKE)
+          )
+        );
+        operatorControllerXButton.onTrue(
+          new SequentialCommandGroup(
+            new MoveArmToPosition(arm, Constants.ARM_INSIDE_ROBOT),
+            new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_HUMAN_PLAYER)
+          )
+        );
+        operatorControllerBButton.onTrue(
+          new SequentialCommandGroup(
+            new MoveArmToPosition(arm, Constants.ARM_INSIDE_ROBOT),
+            new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_MIDDLE)
+          )
+        );
+        operatorControllerAButton.onTrue(new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_INSIDE_ROBOT));
 
         //primary arm
         operatorDPadUp.onTrue(
           new SequentialCommandGroup(
             new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_INSIDE_ROBOT),
             new MoveArmToPosition(arm, Constants.ARM_SCORE_HIGH)
         ));
-
         operatorDPadRight.onTrue(
           new SequentialCommandGroup(
             new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_INSIDE_ROBOT),
             new MoveArmToPosition(arm, Constants.ARM_SCORE_MID)
         ));
-
+        operatorDPadLeft.onTrue(
+          new SequentialCommandGroup(
+            new FloorMoveArmToPostion(floorArm, Constants.FLOOR_ARM_INSIDE_ROBOT),
+            new MoveArmToPosition(arm, Constants.ARM_HUMAN_PLAYER_INTAKE)
+          )
+        );
         operatorDPadDown.onTrue(new MoveArmToPosition(arm, Constants.ARM_INSIDE_ROBOT));
-        
-        //primary intake
-        operatorControllerYButton.whileTrue(new DropCube(intake));
-        operatorControllerAButton.whileTrue(new IntakeCube(intake));
-        operatorControllerXButton.whileTrue(new DropCone(intake));
-        operatorControllerBButton.whileTrue(new IntakeCone(intake));
```

Comment:
Based on the above comment we add back the Intake/Pickup controls, there could be two buttons (left / right bumper):

* Left bumper : 
  * IntakeCone
* Right bumper:
   * Conditional Command:
      * If arm is extended:
        *  IntakeCube
      * If arm is NOT extended:
        * Conditional Command:
          * If floor arm is extended:
             * FloorDropCube
          * If floor arm is NOT extended:
             * Print command "cube pickup is not possible as both arms are stored"